iengine/iengine-app/src/main/java/io/tapdata/flow/engine/V2/node/hazelcast/HazelcastBaseNode.java

```diff
@@ -561,11 +565,36 @@ public synchronized TapCodeException errorHandle(Throwable throwable, String err
 		TapCodeException currentEx = wrapTapCodeException(throwable);
 		TaskDto taskDto = processorBaseContext.getTaskDto();
 		handleWhenTestRun(taskDto, currentEx);
-
+		List<ErrorEvent> errorEvents;
+		if(CollectionUtils.isNotEmpty(taskDto.getErrorEvents())){
+			errorEvents = taskDto.getErrorEvents();
+		}else{
+			errorEvents = new ArrayList<>();
+		}
+
+		List<ErrorEvent> skipErrorEvents = errorEvents.stream().filter(ErrorEvent::getSkip).collect(Collectors.toList());
+		SkipError skipError = SkipErrorStrategy.getSkipErrorStrategy(null).getSkipError();
```

Comment:
最好别写这种传入null的，阅读性比较差，可以加一个getDefaultSkipErrorStrategy()的方法
```suggestion
		SkipError skipError = SkipErrorStrategy.getDefaultSkipErrorStrategy().getSkipError();
```